### DISCUSSION TITLE
We should add some good looking icons in this project

### DISCUSSION THREAD

**Discussion by authorOf** [authorOf/push#8] [Ideas]
### This is the best free icon library in my opinion
Link for the website -> https://www.remixicon.com
![Screenshot from 2023-09-07 18-08-20](https://github.com/authorOf/push/assets/135622087/48597593-070c-4b07-9465-c7e1f8b99162)

**Comment by ghost** [comment]
Great Idea :bulb: :+1:

**Reply by authorOf** [reply]
Thank you :heart:

**Comment by ghost** [comment]
Hey bro! you made some grammar mistake in the title of this discussion

**Reply by authorOf** [reply]
Oh! Sorry for that

**Reply by authorOf** [reply]
Now it looks good ?
